includes/discovery/sensors/states/netbotz.inc.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2017 Søren Friis Rosiak <sorenrosiak@gmail.com> 
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$temp = snmpwalk_cache_multi_oid($device, 'otherStateSensorTable', array(), 'NETBOTZ410-MIB');
+$cur_oid = '.1.3.6.1.4.1.5528.100.4.2.10.1.3.';
+
+if (is_array($temp)) {
+    //Create State Index
+    $state_name = 'otherStateSensorErrorStatus';
```

Comment:
It's not used anywhere and is referenced to the mib so seems appropriate if a little obscure.